src/withTheme.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Form from "./";
+
+function withTheme(data) {
+  return class extends Component {
+    render() {
+      let { templates, widgets, fields, ...restData } = this.props;
```

Comment:
Can you rename `restData` to `otherProps`?

Reply:
Sure, I'm very bad at naming variables.